index.html

```diff
@@ -294,9 +370,25 @@ <h1>To Do List</h1>
     </div>
 
 
-    <div class="myframe">
-        <iframe src="To-Do-List/calendar.html" frameborder="0"></iframe>
```

Comment:
The code is running well on my side. I am glad the merging had no complications. Your all good to go!!!